### ISSUE TITLE
Typescript: missing children on ShadowProps

### ISSUE THREAD

**Issue by Czino** [SrBrahma/react-native-shadow-2#44]
I encountered the following error when using <Shadow>

```Type '{ children: (Element | null)[]; viewStyle: any[]; }' is not assignable to type 'IntrinsicAttributes & ShadowProps'.
  Property 'children' does not exist on type 'IntrinsicAttributes & ShadowProps'.ts(2322)
```

Even for the simplest code:
```
<Shadow>
      <TextInput/>
</Shadow>
```

I could only solve it by adding `children?: ReactNode;` to the `ShadowProps`

**Comment by SrBrahma** [comment]
This is strange.

The `Shadow` component is defined by `export const Shadow: React.FC<ShadowProps> = ...`, and React.FC automatically adds the `children` property to the component type. The ShadowProps type doesn't have it, but does the Shadow.

What version of react-native-shadow-2 and @types/react are you using?

**Comment by SrBrahma** [comment]
I can add the `children?: React.ReactNode;` to the ShadowProps (just saw that ViewProps for example has it), but I don't see how you are having this issue

**Comment by SrBrahma** [comment]
> I believe this issue is likely from the @types/react 18 upgrade:
> 
> https://solverfox.dev/writing/no-implicit-children/

Thanks. Tomorrow I will release a hotfix.

**Comment by SrBrahma** [comment]
Hi! Just released 6.0.4 (https://github.com/SrBrahma/react-native-shadow-2/commit/ea9bfd282bab14be5a3515defeeb113dbc53b463) fixing this issue.

Many thanks for the report, @Czino and @hcharley!
